### ISSUE TITLE
Unable to scan for extended advertising

### ISSUE THREAD

**Issue by Nilon123456789** [adafruit/Adafruit_CircuitPython_BLE#188]
I'm trying to listen for extended advertising from an nrf52833. I tried the [ble_detailed_scan.py](https://github.com/adafruit/Adafruit_CircuitPython_BLE/blob/main/examples/ble_detailed_scan.py) script, by adding the `extended=True` flag. I know that my bluetooth adapter is compatible since I can detect my device in the [BluetoothLEExplorer](https://github.com/microsoft/BluetoothLEExplorer) from microsoft.

**Comment by tannewt** [comment]
What platform are you running this code on?

**Comment by Nilon123456789** [comment]
Windows 10

**Comment by tannewt** [comment]
I don't think this is a bug with this library. I suspect it is an issue with https://github.com/hbldh/bleak which we use underneath blinka bleio.
